gtsfm/evaluation/merge_reports.py

```diff
@@ -0,0 +1,107 @@
+"""Merge two GTSFM reports for side-by-side comparison.
+
+Authors: John Lambert
+"""
+
+import argparse
+from collections import defaultdict
+from typing import Any, Dict, List, Tuple
+
+from tabulate import tabulate
+
+GTSFM_REPORT_TABLES = Dict[str, Dict[str, Any]]
```

Comment:
I'd prefer the name explicitly convey that is is a table coming from a single report.

Also, think we can drop the GTSFM prefix